libs/counterfactuals/src/lib/CounterfactualComponentUtils.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+enum FieldChangeUpdate {
+  Dither = "dither",
+  Property = "property",
+  Type = "type"
+}
+
+export function isJustTypeChange(changedKeys: string[]): boolean {
```

Comment:
I'm confused by the "just" 🙂 Based on the comment below would "hasAxisTypeChanged" be a suitable name?

Reply:
thank you, updated